mozilla-release/browser/base/content/browser.js

```diff
@@ -7396,6 +7399,44 @@ var gRemoteTabsUI = {
   }
 };
 
+// Small utility handler which propagates current tab "privateness" property as
+// available to CSS selectors through #navigator-toolbox[on_private_tab]
+const _gPrivateTabUI = {
```

Comment:
There's some. 
I was too lazy :) https://en.wikipedia.org/wiki/Broken_windows_theory
Will attempt to fix this.